### PR TITLE
Feature/zaakobject polymorphism

### PR DESCRIPTION
**Pull request by annashamray**
**Changes:**
1. Add Zaakobject models
1. Register them in the Admin
1. Add serializers based on them
1. Change Zaakobject serializer to polymorphic 
1. Generate schema with polymorphic Zaakobject resource
1. Write tests

### CODE REVIEW

File: src/zrc/datamodel/models/betrokkene.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+from vng_api_common.fields import BSNField, RSINField
+
+from ..constants import GeslachtsAanduiding, SoortRechtsvorm
+from .base_models import Rol, ZaakObject
+from .zaakobjecten import ZakelijkRechtHeeftAlsGerechtigde
+
+logger = logging.getLogger(__name__)
+
+
+class RolZaakobjectRelationMixin(models.Model):
```
Comment (sergei-maertens) [inline src/zrc/datamodel/models/betrokkene.py]:
```suggestion
class AbstractRolZaakobjectRelation(models.Model):
```

It looks weird to see classes inherit from just a "mixin". This is of course not a mixin (mixins should only add behaviour via methods and possibly some attributes), but an abstract base model.

**Comment by annashamray** [comment]
PR based on https://github.com/VNG-Realisatie/vng-api-common/pull/70 
I'll rebase this branch, but for passing tests `vng_api_common` should be updated

File: src/zrc/datamodel/models/betrokkene.py (hunk 1 above)
Comment (annashamray) [reply src/zrc/datamodel/models/betrokkene.py]: abstract model classes are renamed

**Comment by annashamray** [comment]
Branch is rebased based on develop. After `vng_api_common` is updated tests should pass

**Pull request merged**